stable/wordpress/templates/svc.yaml

```diff
@@ -10,11 +10,17 @@ metadata:
 spec:
   type: {{ .Values.serviceType }}
   ports:
-  - name: http
-    port: 80
-    targetPort: http
-  - name: https
-    port: 443
-    targetPort: https
+    - name: http
+      port: 80
+      targetPort: http
+      {{- if (and (eq .Values.serviceType "NodePort") (not (empty .Values.nodePort.http)))}}
+      nodePort: {{ default "" .Values.nodePort.http }}
```

Comment:
Why do you provide a default parameter if you check in the line before that `nodePort.http` is set?

Reply:
Yes, good point.  I corrected this and retested for lint checks.  Thank you for pointing that out.